test/routes/index-test.js

```diff
@@ -0,0 +1,21 @@
+const { expect } = require('chai')
+const got = require('got');
+
+const { server } = require('../../src/index') 
+
+describe('Index page', () => {
+    before(async () => {
+        await server.start();
+        // console.log(`Server started on ${server.host}:${server.port}`);
+    });
+    
+    after(async () => {
+        await server.stop();
+        // console.log('Server is down')
+    });
+
+    it('should show "Hello Node.js" message', async () => {
+        const response = await got(`http://${server.host}:${server.port}`);
```

Comment:
I would like to see an assert for the response as well. Since different response codes can deliver a very different user experience :smile: